settings/settings.go

```diff
@@ -1,31 +1,26 @@
 package settings
 
-import (
-	"os"
-	"sync"
-)
+import "os"
 
-type Settings map[string]string
+var _settings map[string]string
 
-var settings Settings
-var once sync.Once
-
-func getSetting(settings Settings, key string, defaultValue string) {
+func setSetting(key string, defaultValue string) {
 	if value, present := os.LookupEnv(key); present {
-		settings[key] = value
+		_settings[key] = value
 	} else {
-		settings[key] = defaultValue
+		_settings[key] = defaultValue
 	}
 }
 
-func GetSettings() Settings {
-	once.Do(func() {
-		settings = make(Settings)
-		getSetting(settings, "GO_LAUNCH_A_SURVEY_LISTEN_HOST", "0.0.0.0")
-		getSetting(settings, "GO_LAUNCH_A_SURVEY_LISTEN_PORT", "8000")
-		getSetting(settings, "SURVEY_RUNNER_URL", "http://localhost:5000")
-		getSetting(settings, "JWT_ENCRYPTION_KEY_PATH", "jwt-test-keys/sdc-user-authentication-encryption-sr-public-key.pem")
-		getSetting(settings, "JWT_SIGNING_KEY_PATH", "jwt-test-keys/sdc-user-authentication-signing-rrm-private-key.pem")
-	})
-	return settings
+func init() {
+	_settings = make(map[string]string)
+	setSetting("GO_LAUNCH_A_SURVEY_LISTEN_HOST", "0.0.0.0")
+	setSetting("GO_LAUNCH_A_SURVEY_LISTEN_PORT", "8000")
+	setSetting("SURVEY_RUNNER_URL", "http://localhost:5000")
+	setSetting("JWT_ENCRYPTION_KEY_PATH", "jwt-test-keys/sdc-user-authentication-encryption-sr-public-key.pem")
+	setSetting("JWT_SIGNING_KEY_PATH", "jwt-test-keys/sdc-user-authentication-signing-rrm-private-key.pem")
 }
+
+func GetSetting(name string) string {
```

Comment:
_pretends to be `go lint`_

this stutters when called, e.g. `settings.GetSetting`, could just be `settings.Get`